notification2-examples/hello-world-microservice/README.md

```diff
@@ -0,0 +1,149 @@
+
+As an example, if we post the following measurement from the specified device that meets our filter criteria above:
+
+
+
+
+```json
+{
+  "c8y_SpeedMeasurement": {
+    "T": {
+      "value": 100,
+      "unit": "km/h"
+    }
+  },
+  "c8y_Speed2Measurement": {
+    "T": {
+      "value": 150,
+      "unit": "km/h"
+    }
+  },
+  "c8y_Speed3Measurement": {
+    "T": {
+      "value": 200,
+      "unit": "km/h"
+    }
+  },
+  "c8y_MaxSpeedMeasurement": {
+    "T": {
+      "value": 300,
+      "unit": "km/h"
+    }
+  },
+  "time":"2021-06-11T17:03:14.000+02:00",
+  "source": {"id":"<DEVICE_ID>"},
+  "type": "c8y_Speed"
+}
+```
+we will receive the following message as we have specified to transform the filtered measurement to only include c8y_SpeedMeasurement and c8y_MaxSpeedMeasurement.
+
+```json
+{
+  "c8y_SpeedMeasurement": {
+    "T": {
+      "value": 100,
+      "unit": "km/h"
+    }
+  },
+  "c8y_MaxSpeedMeasurement": {
+    "T": {
+      "value": 300,
+      "unit": "km/h"
+    }
+  },
+  "time":"2021-06-11T17:03:14.000+02:00",
+  "source": {"id":"<DEVICE ID>"},
+  "type": "c8y_Speed"
+}
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+  > Important: The Microservice hosting feature must be activated on your tenant, otherwise your request will return an error message like “security/Forbidden, access is denied”. This feature is not assigned to tenants by default, so trial accounts won’t have it. Contact [product support](https://cumulocity.com/guides/welcome/contacting-support/) so that we can assist you with the activation. Note that this is a paid feature. Microservice SDK for Java - Cumulocity IoT Guides
+- contents of this repository
+
+### Instructions
+1. Create a test user using the Cumulocity Management UI
+    - From the Application Switcher select Administration
+    - In the Navigation pane expand Accounts and go to Users
+    - Click on 'Add user', fill in the form for the new user, untick "Send password reset link as email" and provide a password
+    - Click Save
+    - Enable notification support:
+        - Using Navigation pane, go to Accounts and then Roles, click on "Add global role"
+        - Name the role "Notifications" and from the list of permissions tick "Notification 2" in Admin column
+        - click Save
+        - go to Accounts then Users
+        - in the list, find the username created a few steps before and under "Global roles" open the dropdown
+        - find and tick "Notifications" and click "Apply"
+2. Creating a test device
+    - open the demo directory
+    - edit the `device.py` script and provide the following credentials for your newly created user as well as the platform url:
```

Comment:
```suggestion
    - edit the `device.py` script and provide values for the following variables for your newly created user as well as the platform url:
```

'Variables' might not be right term. I might be wrong, but I don;t see the URL as part of the credentials (maybe it is, as an authority). Maybe 'settings' if there's no better word.